resourcescaler.go

```diff
@@ -48,147 +47,142 @@ func New(kubeconfigPath string, namespace string) (scaler_types.ResourceScaler,
 	}, nil
 }
 
-func (s *AppResourceScaler) SetScale(resource scaler_types.Resource, scaling int) error {
-
-	// get ingress by resource name
-	ingress, err := s.kubeClientSet.ExtensionsV1beta1().Ingresses(s.namespace).Get(string(resource), meta_v1.GetOptions{})
-	if err != nil {
-		s.logger.WarnWith("Failure during retrieval of ingress", "resource_name", string(resource))
-		return errors.Wrap(err, "Failed getting ingress instance")
+func (s *AppResourceScaler) SetScale(resource scaler_types.Resource, scale int) error {
```

Comment:
Looks like scale should be a bool or enum with 2 values, wince this doesn't really support anything else beside to/from zero *by design*

Reply:
Mmm yup, but changing it requires `nuclio` `scaler` and `app-resource-scaler` PR so lets postpone it for now